clock_exercise/BigButton.qml

```diff
@@ -3,16 +3,16 @@ import QtQuick 2.0
 Rectangle {
     id: button
 
-    property bool enabled: true
+    property bool active: true
```

Comment:
Sempre un commento relativo all'atomicità dei commit: secondo me il rename di questa property poteva essere un commit a sé stante, sarebbe stato anche facile da descrivere con qualcosa del tipo `Rename property enabled->active in BigButton.qml`.

Avere dei commit il più possibile atomici è molto importante per una serie di motivi:
* ci sono casi in cui vuoi ricopiare una modifica sola da un branch a un altro: se hai un commit preciso, che riguarda una singola feature questo problema lo risolvi facilmente con il comando `git cherrypick` che ti replica il commit altrove, ma se il tuo commit riguarda parti diverse di codice, allora diventa difficile
* se qualcosa dentro un commit causa problemi di vario tipo (bug, crash, imperfezioni grafiche, etc.) vorrei essere in grado di rimuovere solo e soltanto quelle modifiche problematiche: se il commit è atomico, posso usare `git revert` per annullare le modifiche apportate da un commit precedente, oppure rimuovere il commit in tronco dallo storico usando un rebase, ma se il commit è pieno di modifiche estranee, diventa più difficile gestire la cosa

Per atomico, intendo il fatto di avere modifiche che riguardino il più possibile un singolo set di modifiche, come ad esempio questo del renaming di una property.

Un indicatore del fatto che questo commit poteva essere spezzato, è il fatto che nella descrizione tu abbia una virgola che di fatto descrive quelli che potrebbero essere stati due commit distinti.